python/lsst/sconsUtils/eupsForScons.py

```diff
@@ -38,3 +38,10 @@ def setupEnvNameFor(productName):
     
     utils.setupEnvNameFor = setupEnvNameFor
 
+def getEups():
+    global _eups
```

Comment:
I think the more idiomatic way is to store the singleton as an attribute on the `getEups` function itself (`getattr(getEups, "_eups"`) rather than using a global.